### PR TITLE
Maximus5.ConEmu version 11.220.4180

### PR DESCRIPTION
**Pull request by GNQG**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58088)

### CODE REVIEW

File: manifests/m/Maximus5/ConEmu/11.220.4180/Maximus5.ConEmu.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/Maximus5/ConEmu/releases/download/v22.04.18/ConEmuSetup.220418.exe
+  InstallerSha256: D04E95800D932D522267119CFC4A740C3334FDC9A018C7BB0843B5A69D880206
+  InstallerSwitches:
+    Silent: /p:x64 /quiet /norestart
+    SilentWithProgress: /p:x64 /passive /norestart
```
Comment (Trenly) [inline manifests/m/Maximus5/ConEmu/11.220.4180/Maximus5.ConEmu.installer.yaml:22]:
```suggestion
    Silent: /quiet
    SilentWithProgress: /passive
    Custom: /p:x64 /norestart
```

File: manifests/m/Maximus5/ConEmu/11.220.4180/Maximus5.ConEmu.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  ProductCode: '{70320E43-B75D-4519-ADBA-0C0D0157354D}'
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://github.com/Maximus5/ConEmu/releases/download/v22.04.18/ConEmuSetup.220418.exe
+  InstallerSha256: D04E95800D932D522267119CFC4A740C3334FDC9A018C7BB0843B5A69D880206
+  InstallerSwitches:
+    Silent: /p:x86 /quiet /norestart
+    SilentWithProgress: /p:x86 /passive /norestart
```
Comment (Trenly) [inline manifests/m/Maximus5/ConEmu/11.220.4180/Maximus5.ConEmu.installer.yaml:30]:
```suggestion
    Silent: /quiet
    SilentWithProgress: /passive
    Custom: /p:x86 /norestart
```

**Comment by GNQG** [comment]
After some tests I guess `/p:arch` switch must be first argument of install command.

**Review by OfficialEsco — APPROVED** [review]
> After some tests I guess `/p:arch` switch must be first argument of install command.

Yepyep i also tested the suggested change a good while back, this is a rare case where we should be able to specify the location of the Custom/Upgrade switch

**Pull request merged**